pkg/utils/resource/resource.go

```diff
@@ -31,3 +39,20 @@ func HashOf(resource any) (string, error) {
 	}
 	return fmt.Sprintf("%x", sha256.Sum256(jsonBytes)), nil
 }
+
+// IsObjOversized checks if the given object exceeds the specified size limit in bytes.
+// It returns the number of bytes the object is over the limit (positive value) or
+// the number of bytes remaining before reaching the limit (negative value).
+//
+// This utility is useful in cases where KubeFleet needs to check if it can create/update
+// an object with additional information.
+func IsObjOversized(obj runtime.Object, sizeLimitBytes int) (int, error) {
```

Comment:
the function name suggests that the return type is a boolean. Can you call this `CalculateBytesOverSizeLimit `?

Reply:
Sure, let me make the edit 👌